demo/Collapsible.demo.js

```diff
@@ -5,49 +5,41 @@ const CollapsibleDemo = () => (
   <Fragment>
     <code>
       <pre>
-        {'<Collapsible />'}
-        <br />
-        Props:
-        <br />
-        openOnInit: true
-        <br />
-        transTime: undefined
-        <br />
-        transCurve: undefined
+        {'<Collapsible'}
```

Comment:
Let's revert back to straight text here in an effort to make the demo as skeleton as possible. To achieve the same "nesting" affect as the `<ul><li>` structure solves, just use the `&emsp;` HTML entity [as see here](https://github.com/trouble/react-node-position/blob/master/demo/NodeDetails.js) for single or double tabs.